src/client/ua_client.c

```diff
@@ -50,10 +52,12 @@ UA_Client * UA_Client_new(UA_ClientConfig config) {
 
 static void UA_Client_deleteMembers(UA_Client* client) {
     UA_Client_disconnect(client);
-    UA_Connection_deleteMembers(&client->connection);
-    UA_SecureChannel_deleteMembersCleanup(&client->channel);
-    if(client->endpointUrl.data)
-        UA_String_deleteMembers(&client->endpointUrl);
+    UA_Connection_deleteMembers(client->connection);
```

Comment:
* indention
* first case can be simplified to just a _delete( unstead of calling _deleteMembers+free